### DISCUSSION TITLE
（一个跟改进工具没啥关系的求助）想问下开发者，途中这个部分是不是就是插件的全部了？还是说所有插件在做页面元素的时候都会在这里头显示呢？

### DISCUSSION THREAD

**Discussion by hmrg-grmh** [hcfyapp/crx-selection-translate#1061] [求助]
![image](https://user-images.githubusercontent.com/68635334/119829183-8f16c600-bf2d-11eb-9d7e-bdd7f2ed2847.png)

检查离线网页（用的SingleFile）的代码发现的。

想尽可能精简文件，所以考虑把不是页面重点的部分删一下。因为想到开发者应该对这个熟悉点儿吧，就来问问。

想问的是：删掉图中浅蓝色的那行看起来也没什么问题，这部分在网页里的含义，一般是啥。。。（有空着再理我就成🙈）

**Comment by lmk123** [comment]
是的，蓝色那行就是划词翻译注入的代码块了，这行代码包含了你在网页上看到的翻译窗口。

**Reply by hmrg-grmh** [reply]
我试了一下，好像提前删掉它也不会有什么问题。。。

注意到它是因为，检查离线的网页的时候发现了这样一块站空不小的部分。不管有没有使用翻译。

不过这就是不同工具联合使用的问题了，我现在是只要不翻译又要离线就用火狐的无痕（里面没开多少扩展），反正当初为了一天多用点 DeepL 我搞了仨浏览器……

**Reply by lmk123** [reply]
> 注意到它是因为，检查离线的网页的时候发现了这样一块站空不小的部分。不管有没有使用翻译。

你说的应该是 #1030 这个问题吧
